upload-script

```diff
@@ -10,12 +10,11 @@ if [[ -z "$GITHUB_TOKEN" ]]; then
 fi
 
 # Ensure that there is a pattern specified.
-if [[ -z "$1" ]]; then
+if [[ -n "$2" ]]; then
```

Comment:
This change I don't understand.

With your PR we expect either:

* One argument == pattern.
* Two argumetns == pattern, release ID

So testing for empty $1 is still the right thing to do for exiting if a pattern is missing.

Reply:
I really do apologize, I added a check for my argument and then removed it. I must have copied it over the original by mistake. I'll update this.

Reply:
@skx  I amended my previous commit.